test/integration/006_simple_dependency_tests/test_simple_dependency.py

```diff
@@ -257,6 +257,15 @@ def schema(self):
     def models(self):
         return "models"
 
+    @property
+    def project_config(self):
+        return {
+            'config-version': 2,
+            'models': {
+                '+any_config': "{{ target.name }}",
```

Comment:
This works for the most common cases that we should expect to see. It still doesn't work for one of the configs that we saw in the wild:
```yml
# dbt_project.yml
models:
  +enabled: "{{ target.name in ['redshift', 'postgres'] | as_bool }}"
```
That still raises the error:
```
17:08:37  Encountered an error:
'in <string>' requires string as left operand, not NoneType
```
Is there any way we could hope to handle that? I guess by returning `""` instead of `None` for all values?

Reply:
Seems like it should be okay... I've pushed a change to return an empty string

Reply:
@gshank I see you made the change to use `""` instead of `None`. I think it's the right call. The only way I could think of this breaking is if:
- ~There's a specific node config that accepts `None`, but not `str`~ Confirmed that config-level validation doesn't happen in `deps`/`clean`, it's when we're parsing + validating actual nodes
- A user has defined an expression like:
```yml
models:
  # target.port is numeric when set, but it will be "" in deps/clean
  +query_tag: "{{ target.port + 1234 }}"
```
That doesn't work with `NoneType` either, so I think we're either at parity or strictly better off with a string:
```
can only concatenate str (not "int") to str
unsupported operand type(s) for +: 'NoneType' and 'int'
```

Reply:
Here's where I miss Perl... :). It would have treated the empty string as 0 here.